devtools/common/src/main/java/io/quarkus/generators/rest/BasicRestProjectGenerator.java

```diff
@@ -111,8 +108,12 @@ private void generate(final String templateName, final Map<String, Object> conte
                 throws IOException {
             if (!writer.exists(outputFilePath)) {
                 String path = templateName.startsWith("/") ? templateName : "/" + templateName;
+                InputStream resourceStream = getClass().getResourceAsStream(path);
```

Comment:
Wondering if we should have a try with resources here and properly close the stream?